src/Uuid.php

```diff
@@ -93,7 +94,7 @@ class Uuid implements UuidInterface
     /**
      * Regular expression pattern for matching a valid UUID of any variant.
      */
-    const VALID_PATTERN = '^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}$';
+    const VALID_PATTERN = Validator::VALID_PATTERN;
```

Comment:
@ramsey Since this will be released with 4.0, should we remove this and just have it live on the validator vs UUID as well?